src/views/Languages/index.jsx

```diff
@@ -0,0 +1,297 @@
+import { hot } from 'react-hot-loader';
+import { Component, Fragment } from 'react';
+import Typography from '@material-ui/core/Typography';
+import { Link } from 'react-router-dom';
+import { graphql, withApollo } from 'react-apollo';
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Drawer from '@material-ui/core/Drawer';
+import IconButton from '@material-ui/core/IconButton';
+import Divider from '@material-ui/core/Divider';
+import Hidden from '@material-ui/core/Hidden';
+import MenuIcon from 'mdi-react/MenuIcon';
+import CloseIcon from 'mdi-react/CloseIcon';
+import { memoizeWith } from 'ramda';
+import uniqBy from 'lodash.uniqby';
+import AppBar from '../../components/AppBar';
+import TasksTable from '../../components/TasksTable';
+import Sidebar from '../../components/Sidebar';
+import ErrorPanel from '../../components/ErrorPanel';
+import Spinner from '../../components/Spinner';
+import bugsQuery from '../bugs.graphql';
+import commentsQuery from '../comments.graphql';
+import {
+  BUGZILLA_ORDER,
+  BUGZILLA_PAGE_NUMBER,
+  BUGZILLA_PAGE_SIZE,
+  BUGZILLA_STATUSES,
+  GOOD_FIRST_BUG,
+  BUGZILLA_LANGUAGES,
+  MENTORED_BUG,
+} from '../../utils/constants';
+import extractWhiteboardTags from '../../utils/extractWhiteboardTags';
+
+const bugzillaSearchOptions = {
+  statuses: Object.values(BUGZILLA_STATUSES),
+  order: BUGZILLA_ORDER,
+};
+const bugzillaPagingOptions = {
+  page: BUGZILLA_PAGE_NUMBER,
+  pageSize: BUGZILLA_PAGE_SIZE,
+};
```

Comment:
Since we're using this object in multiple places, we can probably have this in `constant.js` as

```js
BUGZILLA_PAGING_OPTIONS = {
  // ...
}
```